File: Proxy/Python/Etisalat.py
# -*- coding: utf-8 -*-
"""
Created on Mon May 30 20:23:25 2022

@author: Mahmoud Saeed
"""

from ISP import ISP

class Etisalat(ISP):
    browsingSpeed = 10
    
    def serverSite(self , url):
        return "http://{}.com".format(url)
        
    def setspeed(self , speed):
        self.browsingSpeed = speed
     
    def getspeed(self):
        return self.browsingSpeed
    
    

File: Proxy/Java/src/proxy/Etisalat.java
package proxy;

public class Etisalat implements ISP {
    private int  browsingSpeed = 10;

    @Override
    public String serverSite(String url) {
        return String.format("http://%s.com",url);
    }

    public int getBrowsingSpeed() {
        return browsingSpeed;
    }

    public void setBrowsingSpeed(int browsingSpeed) {
        this.browsingSpeed = browsingSpeed;
    }
}
